src/main/java/Task.java

```diff
@@ -0,0 +1,14 @@
+/** Describes generic Task objects. */
+abstract public class Task {
+    protected boolean isDone;
```

Comment:
Would it be better to set default value false for Task so that the subclasses do not require setting it in their respective constructors?

Reply:
good idea, how did i miss that?